features/GEM111_No_duplicated_points_within_a_polyloop_or_polyline.feature

```diff
@@ -0,0 +1,23 @@
+@implementer-agreement
+@GEM
+Feature: GEM111 - No duplicated points within a polyloop or polyline
+
+  Scenario: Agreement on no duplicated points within a polyloop
+
+      Given A file with Schema Identifier "IFC2X3"
+      And An IfcPolyLoop
+      Then The IfcPolyLoop must have no duplicate points
```

Comment:
I'm actually not so sure this is necessary, but we can leave it in. I would assume this is already covered by the Informal Propositions in the schema https://standards.buildingsmart.org/IFC/RELEASE/IFC2x3/TC1/HTML/ifctopologyresource/lexical/ifcclosedshell.htm (but I'm also not 100% sure).

Reply:
This is already covered in the schema with `WR21`: https://standards.buildingsmart.org/IFC/RELEASE/IFC2x3/TC1/HTML/ifctopologyresource/lexical/ifcpolyloop.htm

Reply:
That only seems to check whether there are no points with a different derived Dim attribute from the first point? 

Even if it would, in a formal express rule it would be really hard to factor in the tolerance.

Reply:
My bad, not the `WR21`, its the `UNIQUE` specifier :)

Reply:
I retract my comment, two distinct points could still have same coordinate values ...

Reply:
Correct, I also had to double check to see what it means:

> each element in a list value of this data type shall be different from (i.e., not instance equal to)

instance equal later specified as

> the implementation dependent identifiers are the same

In **8.2.5 Value uniqueness on aggregates** they give two other approaches using the `value_unique()` function or using a custom comparison function.

Reply:
To sum up, are we ok leaving it in?